docs/site/angular-auth-oidc-client/docs/migrations/v11-to-v12.md

```diff
@@ -281,137 +146,318 @@ export const httpLoaderFactory = (httpClient: HttpClient) => {
+  customParamsRefreshTokenRequest?: { [key: string]: string | number | boolean };
+  customParamsEndSessionRequest?: { [key: string]: string | number | boolean };
+  customParamsCodeRequest?: { [key: string]: string | number | boolean };
+  // ...
+}
+```
+
+## Service `OidcSecurityService`
+
+### Getter for `configuration` and object `PublicConfiguration` replaced by method
+
+The getter for the active configuration was removed as well as the object `PublicConfiguration`. As the `AuthWellKnownEndpoints` are now part of the config the property was replaced with methods to get the configuration directly. Please read below how to get the current used configuration.
+
+Old:
+
+```ts
+export class OidcSecurityService {
+  get configuration(): PublicConfiguration {}
 }
 ```
 
+used with
+
+```ts
+const configuration = this.oidcSecurityService.configuration;
+```
+
+New:
+
+```ts
+getConfigurations(): OpenIdConfiguration[] { }
+
+
+getConfiguration(configId?: string): OpenIdConfiguration { }
+```
+
+used with
+
+```ts
+const configuration = this.oidcSecurityService.getConfiguration();
+// or
+const configuration = this.oidcSecurityService.getConfiguration('configId');
+```
+
+### `getToken()` renamed to `getAccessToken()`
+
+Old
+
+```ts
+const accessToken = this.oidcSecurityService.getToken();
+```
+
+New
+
+```ts
+const accessToken = this.oidcSecurityService.getAccessToken();
+```
+
+### `checkAuth()` returning `LoginResponse` instead of boolean
+
+The `checkAuth()` method is now not only returning if you are authenticated or not but instead a complete object containing:
+
+```json
+{
+  isAuthenticated: boolean;
+  userData: any;
+  accessToken: string;
+  idToken: string;
+  configId: string;
+  errorMessage?: string;
+}
+```
+
+Old:
+
+```ts
+this.oidcSecurityService.checkAuth().subscribe((isAuthenticated) => {});
+```
+
+New:
+
+```ts
+this.oidcSecurityService.checkAuth().subscribe(({ isAuthenticated }) => {});
+```
+
+Or
+
+```ts
+this.oidcSecurityService.checkAuth().subscribe(({ isAuthenticated, userData, accessToken, idToken, configId }) => {
+  // ...use data
+});
+```
+
+### `checkAuthIncludingServer()` returning `LoginResponse` instead of boolean
+
+See [`checkAuth()` returning `LoginResponse` instead of boolean](#checkauth-returning-loginresponse-instead-of-boolean).
+
+Same applies for `checkAuthIncludingServer()`
+
+### `forceRefreshSession()` returning `LoginResponse` instead of TokenResponse
+
+See [`checkAuth()` returning `LoginResponse` instead of boolean](#checkauth-returning-loginresponse-instead-of-boolean).
+
+Same return value applies for `forceRefreshSession()`.
+
+Old:
+
+```ts
+this.oidcSecurityService.forceRefreshSession().subscribe((isAuthenticated) => {});
+```
+
+New:
+
+```ts
+this.oidcSecurityService.checkAuth().subscribe(({ isAuthenticated }) => {});
+```
+
+### `authorize(...)` has new `configId` as first parameter
+
+Because multiple configs have been introduce with V12, now the `configId` is the first parameter.

```

Comment:
Minor rewording for clarity:

Because V12 introduces multiple configs, the first parameter is now `configId`.